csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/FileProvider.cs

```diff
@@ -91,7 +91,9 @@ private IEnumerable<FileInfo> SelectSmallFiles(IEnumerable<FileInfo> files)
         private FileInfo[] GetAllFiles()
         {
             logger.LogInfo($"Finding files in {SourceDir}...");
-            var files = SourceDir.GetFiles("*.*", new EnumerationOptions { RecurseSubdirectories = true });
+            var files = SourceDir
```

Comment:
I'm surprised that `GetFiles` is not deterministic?

Reply:
It's a wild guess.